utils/utils.c

```diff
@@ -886,6 +886,23 @@ char *absolute_dirname(const char *path, char *resolved_path)
 	return resolved_path;
 }
 
+char *uftrace_strerror_r(int errnum, char *buf, size_t buflen)
```

Comment:
Can we simply call it `uftrace_strerror`?  The `_r` part in the `strerror_r()` was an ugly addition since it could not replace the original one.